tests/python/relax/frontend/test_onnx_frontend.py

```diff
@@ -0,0 +1,1379 @@
+
+
+# TODO Enable dynamism
+@pytest.mark.parametrize("dynamic", [False])
+def test_skiplayernormalization(dynamic):
+    """test_skiplayernormalization"""
+
+    def verify_skiplayernormalization(input_, skip, gamma, beta, bias):
+        node = onnx.helper.make_node(
+            "SkipLayerNormalization",
+            inputs=["input", "skip", "gamma", "beta", "bias"],
+            outputs=["output", "mean", "std_dev"],
+            domain="com.microsoft",
+        )
+
+        node.attribute.append(onnx.helper.make_attribute("epsilon", 1e-4))
+
+        input_shape = list(input_.shape)
+        skip_shape = list(skip.shape)
+        gamma_shape = list(gamma.shape)
+        beta_shape = list(beta.shape)
+        bias_shape = list(bias.shape)
+        output_shape = list(input_.shape)
+        mean_shape = list([1])
+        std_dev_shape = list([1])
+        if dynamic:
+            input_shape = ["?" for _ in range(len(input_.shape))]
+            skip_shape = ["?" for _ in range(len(skip.shape))]
+            gamma_shape = ["?" for _ in range(len(gamma.shape))]
+            beta_shape = ["?" for _ in range(len(beta.shape))]
+            bias_shape = ["?" for _ in range(len(bias.shape))]
+            output_shape = ["?" for _ in range(len(input_.shape))]
+
+        graph = helper.make_graph(
+            [node],
+            "skiplayernormalization_test",
+            inputs=[
+                helper.make_tensor_value_info("input", TensorProto.FLOAT, input_shape),
+                helper.make_tensor_value_info("skip", TensorProto.FLOAT, skip_shape),
+                helper.make_tensor_value_info("gamma", TensorProto.FLOAT, gamma_shape),
+                helper.make_tensor_value_info("beta", TensorProto.FLOAT, beta_shape),
+                helper.make_tensor_value_info("bias", TensorProto.FLOAT, bias_shape),
+            ],
+            outputs=[
+                helper.make_tensor_value_info("output", TensorProto.FLOAT, output_shape),
+                helper.make_tensor_value_info("mean", TensorProto.FLOAT, mean_shape),
+                helper.make_tensor_value_info("std_dev", TensorProto.FLOAT, std_dev_shape)
+            ],
+        )
+
+        model = helper.make_model(graph, producer_name="skiplayernormalization_test")
+        check_correctness(model, inputs={"input": input_, "skip": skip, "gamma": gamma, "beta": beta, "bias": bias})
+
+    hidden_size = 384
+    batch_size = 4
+    sequence_length = 4
+
+    dtype = "float32"
+    input_array = np.random.random((batch_size, sequence_length, hidden_size)).astype(dtype)
+    skip = np.random.random((batch_size, sequence_length, hidden_size)).astype(dtype)
+    gamma = np.random.uniform(0.5, 0.7, hidden_size).astype(dtype)
+    beta = np.random.randn(hidden_size).astype(dtype) * 0.1
+    bias = np.random.randn(hidden_size).astype(dtype)
+
+    verify_skiplayernormalization(input_array, skip, gamma, beta, bias)
+
+
+def test_embedlayernormalization():
+    """test_embedlayernormalization"""
```

Comment:
```suggestion
```

We shouldn't have docstrings unless they add info

Reply:
I removed the unnecessary docstrings